lib/perl/Genome/Process.pm

```diff
@@ -621,8 +621,9 @@ sub delete {
     }
 
     my $observer;
-    $observer = UR::Context->current->add_observer(aspect=>'commit',
-        callback=>$self->_disk_allocation_cleanup_closure(\$observer));
+    $observer = Genome::Sys::CommitAction->create(
+        on_commit => $self->_disk_allocation_cleanup_closure(\$observer)
```

Comment:
I think we can stop passing `\$observer` into this call.  It looks like it was trying to accomplish the same thing as `once => 1`.